docs/installation.md

```diff
@@ -27,74 +27,85 @@ Here is a command to create and activate a new virtual environment with *conda*
+
+Each of the above installation options are available on PyPI. "Minimal" is also available on conda-forge. (In time, we intend to add more installation options to conda-forge.)
 
 ### 1. "All" Dependencies (excludes some maintainer-only packages)
 
 Use this for `scores` development and general use.
 
 Installs:
-* Mathematical API code and libraries
-* Everything needed to run the tutorial notebooks
-* Testing, static analysis and other developer libraries
-* Does **not** install tools for making packages and releasing new versions
+* Mathematical API code and libraries.
+* Everything needed to run the tutorial notebooks.
+* Testing, static analysis and other developer libraries.
+* Does **not** install tools for making packages and releasing new versions.
 
-#### From a Local Checkout of the Git Repository
+#### With pip
 
 ```bash
+# From a local checkout of the Git repository
 pip install -e .[all]
 ```
 
 ### 2. "Minimal" Dependencies (Mathematical API Functions Only)
 Use this to install the `scores` code into another package or system.
 
 Installs:
-* Mathematical API functions and libraries
+* Mathematical API functions and libraries.
 * Only the required core dependencies. Nothing extra - no tutorials, no developer requirements.
-* (Note for high-performance users - dask is not included by default in the minimal install, but will be used if installed into the environment)
+* (Note for high-performance users - dask is not included by default in the minimal install, but will be used if installed into the environment.)
 
 #### From PyPI
 
 ```bash
 pip install scores
 ```
+#### With conda
+
+```bash
+conda install conda-forge::scores
+```
+
+(Note: at present, only the "minimal" installation option is available from conda. In time, we intend to add more installation options to conda.)
 
 ### 3. "Tutorial" Dependencies
 Use this for running tutorials using `scores`, but when you don't need or want developer tools.
 
 Installs:
-* Mathematical API functions and libraries
-* JupyterLab, Plotly, and libraries for reading data, so that the tutorial notebooks can be run
+* Mathematical API functions and libraries.
+* JupyterLab, Plotly, and libraries for reading data, so that the tutorial notebooks can be run.
 
-#### From a Local Git Repository
+#### With pip 
 
 ```bash
+# From a local checkout of the Git repository
 pip install .[tutorial]
 ```
 
 ### 4. "Maintainer" Dependencies
 Use this to build the docs, create packages and prepare new versions for release.
 
 Installs:
-* Mathematical API functions and libraries
-* Dependencies for building new versions of the `scores` package
-* Dependencies for building the documentation as HTML
+* Mathematical API functions and libraries.
+* Dependencies for building new versions of the `scores` package.
+* Dependencies for building the documentation as HTML.
 
-#### From a Local Git Repository
+#### With pip
 
 ```bash
+# From a local checkout of the Git repository
 pip install -e .[maintainer]
```

Comment:
I think there needs to be one additional step to install `pandoc` separately from pip via an OS package manager

Reply:
Yes, good point. I'll put forward something but basically it should say something along the lines of "people wishing to build the documentation locally will need to install pandoc themselves, such as through their operating system package manager".

Reply:
Thanks for the feedback @nicholasloveday. @tennlee is going to address this in a later PR (he wants to check a few things first).